### ISSUE TITLE
Lemmy 0.19.3 issues

### ISSUE THREAD

**Issue by pkrasicki** [db0/div0-theme#9]
Hi, I found your theme by a pure coincidence and it seems to be based in some way on mine. You might want to know that there is an issue introduced by Lemmy 0.19.x in both our themes, which causes tables to have black text color:
https://github.com/pkrasicki/lemmy-modern-ui-theme-addon/issues/8

![dark-table-text](https://github.com/db0/div0-theme/assets/49721209/53c5969c-5b32-42a7-b42a-c37a61649753)

There is also a bug in Lemmy, which causes code blocks to have a white background when user's system theme is set to light theme:

![lemmy-code-blocks-bug3](https://github.com/db0/div0-theme/assets/49721209/f3ef06e1-cd73-4293-a7d6-a75840bff78d)

I've created a new release, which fixes those issues, but you can probably fix this by adding this CSS code to your theme:
```
/* fix for 0.19.3 tables */
.table
{
	--bs-table-color: #fff;
	--bs-table-hover-color: #fff;
}

/* fix for 0.19.3 code blocks */
code.hljs
{
	color: #99a0ac;
	background: #171717;
	filter: brightness(1.5) saturate(0.8);
}
```

If you use my values, the code blocks will look a bit different than default:
![code-blocks-fix3](https://github.com/db0/div0-theme/assets/49721209/0f1482e3-9a99-4d1b-9806-3e43bc96980a)

**Comment by db0** [comment]
@RandomLegend want to push then relevant patch?

**Comment by RandomLegend** [comment]
fixed in #12
